.github/workflows/build-and-test.yml

```diff
@@ -102,7 +102,7 @@ jobs:
         uses: gradle/gradle-build-action@v2
 
       # Run tests
-      - name: Run Library, Service, and Test Harness tests with coverage
+      - name: Run Library and Service with coverage
```

Comment:
Put "tests" back in so that it is clear this is the step that runs the unittests.